bitnami/mongodb/values-production.yaml

```diff
@@ -797,7 +984,7 @@ volumePermissions:
 ## Prometheus Exporter / Metrics
 ##
 metrics:
-  enabled: true
+  enabled: false
```

Comment:
do not change the default values

Reply:
done

Reply:
This is not done

Reply:
so metrics is the only default variable that I cannot yet figure out how to set it to true because it needs the TLS values somehow. When set to false of course it runs with no issues but once set to true it will not run due to the TLS enabled on the cluster. 
I'll have to look at this more. In the meantime everything else works.

Reply:
The TLS must be disabled by default, since it will be changing the deployment and it could create issues upgrading the chart from previous versions

Reply:
no problem